customize_image.sh

```diff
@@ -37,6 +39,35 @@ if [ $$ != 2 ] && [ -x /usr/bin/newpid ] ; then
 	exec /usr/bin/newpid "$0" "$@"
 fi
 
+# set MINIMG as 0: build the normal image by default
+MINIMG=0
+
+# get the options
+MYOPTS=`getopt -o mh --long minimize,help -- "$@"`
+if [ "$?" != 0 ]; then  usage; exit 1; fi
+eval set -- "$MYOPTS"
+
+# extract options and their arguments into variables.
+while true ; do
+	case "$1" in
+		-m|--minimize) MINIMG=1 ; shift ;;
+		-h|--help) usage ; exit 0;;
+		*) shift; break ;;
+	esac
+done
+
+if [ "$MINIMG" != "1" ]; then
+	echo "The normal image will be built."
+else
+	echo "The lite image will be built."
+fi
+
+if [ "$#" != 1 ] ; then
+	echo 1>&1 "Error: Cann't get path of <image> from parameter"
```

Comment:
Cann't?

Also this seems like a different change, so do a separate commit for this.

Reply:
Cann't => Can't thanks for the catch.

This was on the top to check the number of parameters to see whether path of image available (although it not a strict check). when check failed, the usage mesage would be given.   In this commit, as the number of parameters get manipulated by "getopt and set --", this check should be placed after that.  And it is not proper to just print the usage message, so I add the reason to it, and use the new added usage() function to show the usage message.   I see that they are somewhat integrated and therefor let them in one commit.